src/rules/__tests__/use-deprecated-from-deprecated.test.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const {RuleTester} = require('eslint')
+const rule = require('../use-deprecated-from-deprecated')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+})
+
+ruleTester.run('use-deprecated-from-deprecated', rule, {
+  valid: [],
+  invalid: [
+    // Single deprecated import
+    {
+      code: `import {Tooltip} from '@primer/react'`,
+      output: `import {Tooltip} from '@primer/react/deprecated'`,
+      errors: ['Import deprecated components from @primer/react/deprecated'],
+    },
+
+    // Single deprecated import with existing deprecated entrypoint
+    {
+      code: `import {Tooltip} from '@primer/react'
+import {Dialog} from '@primer/react/deprecated'`,
+      output: `\nimport {Dialog, Tooltip} from '@primer/react/deprecated'`,
+      errors: ['Import deprecated components from @primer/react/deprecated'],
+    },
+
+    // Multiple deprecated imports
+    {
+      code: `import {Dialog, Tooltip} from '@primer/react'`,
+      output: `import {Dialog, Tooltip} from '@primer/react/deprecated'`,
+      errors: ['Import deprecated components from @primer/react/deprecated'],
+    },
+
+    // Mixed deprecated and non-deprecated imports
+    {
+      code: `import {Button, Tooltip} from '@primer/react'`,
+      output: `import {Button, } from '@primer/react'
```

Comment:
Is this `,` after the Button expected? Seems like tests are succeeding so maybe it is expected. If so, are we relying on prettier to remove the comma after the fix?

Reply:
Yeah, let me know if you have a good way to remove it 😅 I couldn't really figure it out. Definitely relying on prettier for this